### PR TITLE
working timeline lookup using name and id

### PR DESCRIPTION
**Pull request by Jshen18**

### CODE REVIEW

File: client/src/TimelineLookUp.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import propTypes from 'prop-types';
+
+const TimelineLookUp = props => (
```
Comment (angelahuang89) [inline client/src/TimelineLookUp.jsx]: Should we destructure props?

File: client/src/index.jsx [hunk 2]
```diff
@@ -48,6 +51,7 @@ class App extends React.Component {
 
   onSubmit() {
     this.setState({ timelineId: shortid.generate() }, () => {
+      console.log('hello on submit button');
```
Comment (angelahuang89) [inline client/src/index.jsx:54]: Let's take out this console log :)

File: client/src/index.jsx [hunk 3]
```diff
@@ -73,10 +77,10 @@ class App extends React.Component {
       createEventDay: e.target.value,
     });
   }
-
   getTrip() {
-    axios.get(`timeline/${this.state.timelineName}/${this.state.timelineId}`)
+    axios.get(`/timeline/${this.state.timelineName}/${this.state.timelineId}`)
```
Comment (angelahuang89) [inline client/src/index.jsx:81]: Do we need this slash?

File: client/src/index.jsx [hunk 4]
```diff
@@ -73,10 +77,10 @@ class App extends React.Component {
...
       .then(({ data }) => {
+        console.log('data', data);
```
Comment (angelahuang89) [inline client/src/index.jsx:83]: No need for this console log anymore!

File: server/index.js [hunk 5]
```diff
@@ -16,6 +16,7 @@ app.use(express.static(`${__dirname}/../client/`));
 app.options('/', (request, response) => response.json('GET,POST,PUT,GET'));
 
 app.get('/timeline/:timelineName/:timelineId', (request, response) => {
+  console.log('request params ID', request.params.timelineId);
```
Comment (angelahuang89) [inline server/index.js:19]: Let's take this out too :)

File: client/src/index.jsx (hunk 3 above)
Comment (andino93) [reply client/src/index.jsx:81]: Yeah the slash is needed for routes

File: client/src/index.jsx (hunk 4 above)
Comment (andino93) [reply client/src/index.jsx:83]: Can you make then a catch instead? `.catch(err => console.error(err))`

**Review by andino93 — APPROVED** [review]

**Pull request merged**